build.properties

```diff
@@ -66,14 +66,12 @@ db.password=dspace
 #db.username=dspace
 #db.password=dspace
 
-# Schema name - if your database contains multiple schemas, you can avoid problems with
-# retrieving the definitions of duplicate object names by specifying
-# the schema name here that is used for DSpace by uncommenting the following entry
-
-# NOTE: this configuration option is for PostgreSQL only. For Oracle, schema is equivalent
-# to user name. DSpace depends on the PostgreSQL understanding of schema. If you are using
-# Oracle, just leave this this value blank.
-
+# Schema name - if your database contains multiple schemas, you can avoid
+# problems with retrieving the definitions of duplicate object names by
+# specifying the schema name that is used for DSpace.
+# ORACLE USAGE NOTE: In Oracle, schema is equivalent to "username". This means
+# specifying a "db.schema" is often unnecessary (i.e. you can leave it blank),
+# UNLESS your Oracle DB Account (in db.username) has access to multiple schemas.
```

Comment:
I suggest adding one more line that says "# For Oracle, it is safe to make db.schema the same as db.username,  doing so will avoid future complications if another schema is ever made available to db.username. Database privileges can be tricky, it is best to be clear which schema we wish to use."